### PR TITLE
add disarm endpoint documentation

### PR DESCRIPTION
**Pull request by mseld**

### CODE REVIEW

File: maas-pro.md [hunk 1]
```diff
@@ -751,32 +810,35 @@ StatusInternalServerError| |
...
+| type         | string  | For vehicles the type will always be "vehicle"                                                 | required |
+| vehicle_type | string  | The vehicles type , values are `scooter`, `bicycle`                                            | required |
+| batteryLevel | integer | The state of charge of the vehicles' battery in percent (0-100)                                | required |
+| location     | object  | The vehicle’s location                                                                         | required |
+| longitude    | number  | The longitude component of the location                                                        | required |
+| latitude     | number  | The latitude component of the location                                                         | required |
+| code         | string  | The 4-letter alphanumeric vehicle code, also available on the physical vehicle and as QR-code. | required |
+| locked       | string  | The vehicle lock state                                                                         | required |
```
Comment (dmitrii-maskevich) [inline maas-pro.md:823]: string?

File: maas-pro.md [hunk 2]
```diff
@@ -835,13 +899,15 @@ On each scooters handlebars, the code, 4 alphanumeric characters, are written in
...
 ```
+
 > A get vehicle by code response.
 
 ```shell
 {
   "data": [
       {
          "type": "vehicle",
+        "vehicle_type": "scooter",
```
Comment (mirzakhany) [inline maas-pro.md:910]: small indent issue in this json sample

File: maas-pro.md (hunk 1 above)
Comment (mseld) [reply maas-pro.md]: sorry it's `bool`

File: maas-pro.md (hunk 2 above)
Comment (mseld) [reply maas-pro.md]: done

**Review by dmitrii-maskevich — APPROVED** [review]

**Pull request merged**